docs/src/GettingStarted/Installation.md

```diff
@@ -2,9 +2,8 @@
 
 ## Install Julia
 
-The current release of the `ClimateMachine` is verified to work with
-Julia 1.3.1. Download it for your platform from [Julia's old
-releases](https://julialang.org/downloads/oldreleases/#v131_dec_30_2019).
+The Climate Machine (CLIMA) uses the Julia programming language and has been tested for the Julia version 1.4.1, which can be downloaded via your package manager or from the [Julia website](https://julialang.org/downloads/#current_stable_release).
+CLIMA has also been tested for Julia version 1.3, which can be found in [Julia's old releases](https://julialang.org/downloads/oldreleases/).
```

Comment:
Do we need to mention this going forward (as we won't actually test against this version any more)?